src/liblsquic/lsquic_full_conn.c

```diff
@@ -1297,9 +1133,9 @@ static int
 process_saved_ack (struct full_conn *conn, int restore_parsed_ack)
 {
     struct ack_info *const acki = conn->fc_pub.mm->acki;
-    struct lsquic_packno_range range;
-    unsigned n_ranges, n_timestamps;
-    lsquic_time_t lack_delta;
+    struct lsquic_packno_range range = { 0 };
```

Comment:
I see why a compiler may require initialization, but in this case the compiler is wrong.  Maybe make it initialize on Windows only?

Reply:
These are warnings. I can disable them if you like.